gradle/shipkit.gradle

```diff
@@ -0,0 +1,40 @@
+shipkit {
```

Comment:
Shouln'd there be a license setting?

Reply:
No, not for the Shipkit configuration, but there's one a few lines below for Bintray.